recipes/pyevtk/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.1.1" %}
+
+package:
+    name: pyevtk
+    version: {{ version }}
+
+source:
+  url: https://github.com/pyscience-projects/pyevtk/archive/v{{version}}.tar.gz
+  sha256: 0f576dc06dc8789970b8d6d0155e7ec1a62d5e60d86c7927ff83f7c4883c5c81
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+    - pytest-runner
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  requires:
+    - pytest >=3.3
+
+  imports:
+    - pyevtk
+    - pyevtk.hl
+
+about:
+  home: https://github.com/pyscience-projects/pyevtk
+  license: BSD 2-clause
```

Comment:
You need a license file entry.

Reply:
Done: https://github.com/conda-forge/staged-recipes/pull/7718/commits/3e03c1d2c1d409b78c74b5bc5613914b92871fdd